cmd/registry/main.go

```diff
@@ -262,3 +258,22 @@ func debugServer(addr string) {
 		log.Fatalf("error listening on debug interface: %v", err)
 	}
 }
+
+type reader func(path string) ([]byte, error)
+
+func createCertPool(certs []string, fileReader reader) (*x509.CertPool, error) {
```

Comment:
Can you give me an usecase where we would use a different reader other than `ioutil.ReadFile`?